data/core/terrain.cfg

```diff
@@ -2721,7 +2758,7 @@ For those who go by land or sea, a bridge is the best of both worlds — for gam
     id=fungus
     name= _ "Fungus"
     editor_name= _ "Fungus"
-    string=Uft
+    string=Tt
```

Comment:
This seems to undefine the `Uft` terrain code. Is that so? Is there a backward compatibility problem here?

Reply:
The `Uft` code isn't a code used on maps, it's the "archetype" used only as a base for other terrains and to provide general information about that archetype (like the sidebar icon and help text). As you can see, it's marked `hidden`, and furthermore it doesn't actually have any graphics, so unless someone manually edited it into their map, there shouldn't be any issues.

Reply:
If someone's cut&pasted terrain definitions in to their UMC, then those terrains might be using Uft as a base. In 1.14, it seems that all terrain aliases use only the virtual terrains as their bases, and I think the help pages start doing odd things if you don't (as in, the game creates a help section for every terrain used as a base, even if it's not a virtual terrain).

Reply:
And wmllint could add rules to auto-update terrain definitions that alias to `Uft`,